src/Illuminate/Support/Manager.php

```diff
@@ -10,7 +10,7 @@ abstract class Manager
     /**
      * The application instance.
      *
-     * @var \Illuminate\Contracts\Foundation\Application
+     * @var \Illuminate\Contracts\Container\Container|\Illuminate\Contracts\Foundation\Application
```

Comment:
Surely this should **only** be `\Illuminate\Contracts\Container\Container`?

Reply:
not sure if we are going that route since I don't have the full knowledge on how 3rd party package using it.

Reply:
Good find! I think it does make sense to fully replace it with the container contract because this is indeed all about resolving objects.

Reply:
I've implemented this here: https://github.com/laravel/framework/pull/29667